Description: automake1.14 warns about 'subdir-objects' not set
together with -Werror this leads to a build failure
Origin: vendor
Bug: http://bugs.debian.org/725534
Forwarded: no
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2013-10-26

--- a/configure.ac
+++ b/configure.ac
@@ -3,7 +3,7 @@

 AC_PREREQ(2.65)
 AC_INIT([Gummi], [0.6.5], [alexvandermey@gmail.com], [gummi], [http://gummi.midnightcoding.org/])
-AM_INIT_AUTOMAKE([foreign -Wall -Werror])
+AM_INIT_AUTOMAKE([foreign -Wall -Werror subdir-objects])

 GETTEXT_PACKAGE=gummi
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE", [The gettext domain])
